src/Api/Middleware/EventLogContextMiddleware.cs

```diff
@@ -13,7 +13,14 @@ public class EventLogContextMiddleware
 
     public async Task InvokeAsync(HttpContext context, IEventLogContext eventLogContext, IFeatureContextProvider featureContextProvider)
     {
+        if (!context.Items.ContainsKey("__AuthorizationMiddlewareWithEndpointInvoked"))
```

Comment:
Would benefit from a comment why this is done. Is there any other "public" way to check if this is an authorized endpoint, `__AuthorizationMiddlewareWithEndpointInvoked` feels like a magic internal string. But I may be wrong?

Reply:
@abergs  see comment :-) !